avocado/utils/ssh.py

```diff
@@ -248,3 +248,28 @@ def copy_files(self, source, destination, recursive=False):
             return result.exit_status == 0
         except process.CmdError as exc:
             raise NWException("failed to copy file {}".format(exc))
+
+    def check_connection(self):
```

Comment:
the method says "check_connection" but you are doing more than that. The method name should be renamed to reflect the code. Something like "remove_control_path()"